src/nvim/highlight_group.c

```diff
@@ -697,19 +697,11 @@ int load_colors(char *name)
   size_t buflen = strlen(name) + 12;
   char *buf = xmalloc(buflen);
   apply_autocmds(EVENT_COLORSCHEMEPRE, name, curbuf->b_fname, false, curbuf);
-  snprintf(buf, buflen, "colors/%s.vim", name);
-  int retval = source_runtime(buf, 0);
+  snprintf(buf, buflen, "colors/%s.*", name);
+  int retval = source_runtime_vim_lua(buf, 0);
   if (retval == FAIL) {
-    snprintf(buf, buflen, "colors/%s.lua", name);
-    retval = source_runtime(buf, 0);
-  }
-  if (retval == FAIL) {
-    snprintf(buf, buflen, "colors/%s.vim", name);
-    retval = source_runtime(buf, DIP_NORTP + DIP_START + DIP_OPT);
-  }
-  if (retval == FAIL) {
-    snprintf(buf, buflen, "colors/%s.lua", name);
-    retval = source_runtime(buf, DIP_NORTP + DIP_START + DIP_OPT);
+    snprintf(buf, buflen, "colors/%s.*", name);
+    retval = source_runtime_vim_lua(buf, DIP_NORTP + DIP_START + DIP_OPT);
```

Comment:
Combine these two calls into one call with `DIP_START + DIP_OPT`

Reply:
What about `DIP_NORTP`? I don't fully understand what that flag does.

Reply:
Not needed anymore.